### PR TITLE
Add Avax receipts logic

### PR DESCRIPTION
**Pull request by antisaa**
Modifying the receipts exporter to support the chains that do not have API method to export all receipts from a block, but have method to get receipt from transaction hash.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,8 +91,8 @@ const fetchEthBlockTimestamps = (fromBlock, toBlock) => {
 
 async function getReceiptsForBlocks(fromBlock, toBlock) {
   logger.info(`Fetching blocks ${fromBlock}:${toBlock}`)
-  const receipts = await fetchEthReceipts(fromBlock, toBlock)
   const blocks = await fetchEthBlockTimestamps(fromBlock, toBlock)
+  const receipts = await fetchEthReceipts(fromBlock, toBlock, blocks)
```
Comment (spiderjako) [inline index.js]: Is there a difference in swapping the lines?

**Review by spiderjako — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (antisaa) [reply index.js]: yes, we have to pass blocks in this function because we need to fetch the transaction hashes

File: helper.js [hunk 2]
```diff
@@ -26,9 +26,15 @@ const setReceiptsTimestamp = (receipts, timestamps) => {
   return collection.forEach(receipts, receipt =>  receipt['timestamp'] = timestamps[receipt.blockNumber])
 }
 
-const parseReceipts = (responses) => {
+const parseReceipts = (responses, avax) => {
```
Comment (YordanPavlov) [inline helper.js]: I don't think we manage to reuse anything here, the avax case seems like a separate one. I would have putted it as a separate method.

File: index.js [hunk 3]
```diff
@@ -36,19 +37,40 @@ const jayson = require('jayson/promise');
...
+const fetchEthReceipts = (fromBlock, toBlock, blocks) => {
   var batch = [];
-  for (fromBlock; fromBlock < toBlock + 1; fromBlock++) {
-    batch.push(
-      parityClient.request(
-        GET_RECEIPTS_ENDPOINT,
-        [web3.utils.numberToHex(fromBlock)],
-        undefined,
-        false
+  if(!AVAX)
```
Comment (YordanPavlov) [inline index.js]: Again this makes the function long without reusing anything.

File: helper.js (hunk 2 above)
Comment (antisaa) [reply helper.js]: fixed

File: index.js (hunk 3 above)
Comment (antisaa) [reply index.js]: fixed

**Review by YordanPavlov — APPROVED** [review]

**Pull request merged**